resources/pvc.py

```diff
@@ -0,0 +1,67 @@
+"""
+General PVC object
+"""
+import tempfile
+import logging
+
+from resources.ocs import OCS
+
+log = logging.getLogger(__name__)
+
+
+class PVC(OCS):
+    """
+    A basic PersistentVolumeClaim kind resource
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initializer function
+        kwargs:
+            See parent class for kwargs information
+        """
+        super(PVC, self).__init__(**kwargs)
+        self.temp_yaml = tempfile.NamedTemporaryFile(
+            mode='w+', prefix='PVC_', delete=False
+        )
+
+    def get_status(self):
+        """
+        Returns the PVC status
+
+        Returns:
+            str: PVC status
+        """
+        return self.data.get('status').get('phase')
+
+    def get_backed_pv(self):
+        """
+        Returns the backed PV name of pvc_name in namespace
+
+        Returns:
+            str: PV name
+        """
+        return self.data.get('spec').get('volumeName')
+
+    def get_size(self):
```

Comment:
same here as well, making this as @property so that access wil be pvc.size

Reply:
done

Reply:
new commit not yet pushed, I guess.

Reply:
@shylesh can I take care of those comments in next PR?
I totally messed up my branch with a scale test case and don't want to block this PR